test/fuzz/transformation_add_bit_instruction_synonym_test.cpp

```diff
@@ -937,7 +937,7 @@ TEST(TransformationAddBitInstructionSynonymTest, NoSynonymWhenBlockIsDead) {
       MakeDataDescriptor(166, {}), MakeDataDescriptor(39, {})));
 }
 
-TEST(TransformationAddBitInstructionSynonymTest, DISABLED_DifferentSingedness) {
+TEST(TransformationAddBitInstructionSynonymTest, ENABLED_DifferentSingedness) {
```

Comment:
You can just rename this to `DifferentSingedness` to enable the test.

Also, please add `OpBitwiseOr`, `OpBitwiseXor` and `OpBitwiseAnd` instructions (with operands with different signedness) to this test as well.

Reply:
Done.